dist/images/Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN mkdir -p /opt/cni/bin
 # Built in ../../go_controller, then the binaries are copied here.
 # put things where they are in the rpm
 RUN mkdir -p /usr/libexec/cni/
-COPY ovn-k8s-overlay ovnkube ovn-kube-util /usr/bin/
+COPY ovn-k8s-cni-overlay ovnkube ovn-kube-util /usr/bin/
```

Comment:
Why do we need ovn-k8s-cni-overlay in /usr/bin? I think ovn-k8s-overlay was an old utility that we no longer build.

Reply:
I would just delete it in 2 dockerfiles.

Reply:
@shettyg I agree.